src/include/iscsi-client/helps.rb

```diff
@@ -110,8 +112,9 @@ def initialize_iscsi_client_helps(include_target)
         "discovery"      => _("<h1>iSCSI Initiator</h1>") +
           _(
             "Enter the <b>IP Address</b> of the iSCSI target server.\n" +
-              "Only change <b>Port</b> if needed. For authentication, use <b>Username</b> and <b>Password</b>. If you do not need authentication,\n" +
-              "select <b>No Authentication</b>.\n"
+              "Only change <b>Port</b>. If you do not need authentication,\n" +
+              "select <b>No Discovery Authentication</b>. " +
+               use_bidirectional_auth
```

Comment:
this won't work. It cannot translate this text. what you need to use is to mark for translation above string and add + after `_()`